### ISSUE TITLE
[NEW ICON] More db-related icons - Oracle, MSSQL, T-SQL, PL/SQL, etc.

### ISSUE THREAD

**Issue by SirCypkowskyy** [tandpfun/skill-icons#409]
### Skill Name

Oracle (SQL), Microsoft SQL Server, Transact-SQL, PL/SQL and more

### Why?

Although the various SQL languages are quite similar, they include some differences. In addition, extensions like Transact SQL or PL/SQL significantly extend the capabilities offered by these languages, hence I believe that each deserves its own icon.

**Comment by FabianAronsson** [comment]
I agree. There should be more icons for the different SQL languages.

**Comment by mertssmnoglu** [comment]
Agreed. There are so many db-query languages.

**Comment by Juan-Carlos-Estevez-Vargas** [comment]
Hello, I am interested in contributing to this repository, could you please give me access?

**Comment by Banovvv** [comment]
I agree as well - Microsoft SQL Server and Transact-SQL are valued skills and should be on the list of available icons.

**Comment by yildizmy** [comment]
Apparently both of them have not been added yet. What is the problem with these icons?

**Comment by RushilJalal** [comment]
> Hello, I am interested in contributing to this repository, could you please give me access?

You have to read the contributing guide for this repo.

**Comment by LelouchFR** [comment]
@SirCypkowskyy @09116168594me @FabianAronsson @mertssmnoglu @Juan-Carlos-Estevez-Vargas @Banovvv @yildizmy @RushilJalal   If you want to contribute, [here](https://github.com/lelouchfr/skill-icons) is the newest version of the project because the maintainer has not showed up in 3 months...
